app/v1/connectors/RetrieveLiabilitiesConnector.scala

```diff
@@ -31,11 +31,22 @@ class RetrieveLiabilitiesConnector @Inject()(val http: HttpClient,
 
   def retrieveLiabilities(request: LiabilityRequest)(implicit hc: HeaderCarrier,
                                                      ec: ExecutionContext): Future[DesOutcome[LiabilityResponse]] = {
+
     import v1.connectors.httpparsers.StandardDesHttpParser._
     implicit val requestFromDate: String = request.from
```

Comment:
what is the purpose of **requestFromDate**

Reply:
it's required for the filtering in the reads

Reply:
I think the filter is based on request **To** date not **from**

Reply:
Done